normandy/control/static/control/js/components/RecipePreview.jsx

```diff
@@ -1,7 +1,8 @@
-import React from 'react'
-import classNames from 'classnames'
-import composeRecipeContainer from './RecipeContainer.jsx'
-import {runRecipe} from '../../../../../selfrepair/static/js/self_repair_runner.js';
+import React from 'react';
+import classNames from 'classnames';
+import composeRecipeContainer from './RecipeContainer.jsx';
+import { runRecipe } from '../../../../../selfrepair/static/js/self_repair_runner.js';
+import NormandyDriver from '../../../../../selfrepair/static/js/normandy_driver.js';
```

Comment:
Nice. That looks like a great idea. We'll just have to make sure none of the places we want to import from collide with any package names.